docs/api/ref/GetAgent.md

```diff
@@ -0,0 +1,224 @@
+    // Failed to connect
+}
+```
+
+</TabItem>
+</Tabs>
+
+The  `getAgent()` function allows web applications to retrieve an FDC3 Desktop Agent API interface to work with, whether they are running in an environment that supports a Desktop Agent Preload (a container-injected API implementation) or a Desktop Agent Proxy (a Browser-based Desktop Agent running in another window or frame). The behavior of `getAgent()` is defined by the [FDC3 Web Connection Protocol (WCP)](../specs/webConnectionProtocol) and communication with a Desktop Agent Proxy in a web-browser is defined by the [Desktop Agent Communication Protocol (DACP)](../specs/desktopAgentCommunicationProtocol). Hence, it allows applications to be written that will work in either scenario without modification or the inclusion of vendor-specific libraries.
+
+If no Desktop Agent is found, a failover function may be supplied by app allowing it to start or otherwise connect to a Desktop Agent (e.g. by loading a proprietary adaptor that returns a `DesktopAgent` implementation or by creating a window or iframe of its own that will provide a Desktop Agent Proxy).
+
+The definition of the `getAgent()` function is as follows:
+
+```ts
+type GetAgentType = ( 
+  params?: GetAgentParams,  
+) => Promise<DesktopAgent>; 
+```
+
+A small number of arguments are accepted that can affect the behavior of `getAgent` and to provide a fallback in case a Desktop Agent is not found, allowing the application to start its own agent or use another mechanism (such a proprietary adaptor) to connect to one.
+
+```ts
+/** 
+ * @typedef {Object} GetAgentParams Type representing parameters passed to the 
+ * getAgent function. 
+ * 
+ * @property {string} identityUrl The app's current URL is normally sent to 
+ * a web-based desktop agent to help establish its identity. This property 
+ * may be used to override the URL sent (to handle situations where an app's 
+ * URL is not sufficiently stable to use for identity purposes). The URL set 
+ * MUST match the origin of the application (scheme, hostname, and port) or  
+ * it will be ignored. If not specified, the app's current URL will be used.  
+ * 
+ * @property {number} timeout Number of milliseconds to allow for an fdc3  
+ * implementation to be found before calling the failover function or 
+ * rejecting (default 750). Note that the timeout is cancelled as soon as a 
+ * Desktop Agent is detected. There may be additional set-up steps to perform 
+ * which will happen outside the timeout. 
+ * 
+ * @property {boolean} channelSelector Flag indicating that the application  
+ * needs access to a channel selector UI (i.e. because it supports User Channels 
+ * and does not implement its own UI for selecting channels). Defaults to true. 
+ * MAY be ignored by Desktop Agent Preload (container) implementations. 
+ * 
+ * @property {boolean} intentResolver Flag indicating that the application  
+ * needs access to an intent resolver UI (i.e. because it supports raising one
```

Comment:
The opening bracket at "(i.e. because" does not have a corresponding closing bracket at the end of the sentence.

Reply:
Fixed in be131b655bbc4812c9cfec080fb8ee86ac81cb85